### PR TITLE
fix: Chips overflowing text field

### LINKED ISSUE #1135: Chips overflowing text field
From https://github.com/mozilla-frontend-infra/balrog-ui/pull/93#discussion_r305955211.

Screen Shot 2019-07-22 at 1 50 54 PM

### PR DESCRIPTION
**Pull request by CREDO23**
#### Changes proposed:
- Add the wrap property to the `chipList` so elements in the `chipList` will wrap instead of overflowing.
- Align all inputs at the same position (`end`)

Frame 107 (2)

@gabrielBusta @bhearsum

### CODE REVIEW

File: ui/src/components/AutoCompleteText/index.jsx [hunk 1]
```diff
@@ -184,7 +187,7 @@ function AutoCompleteText({
                 undefined
               ),
               endAdornment: (
-                <div>
+                <div className={classes.endAdornment}>
```
Comment (bhearsum) [inline ui/src/components/AutoCompleteText/index.jsx:190]: What is this accomplishing? I don't see any visual differences with or without it.

File: ui/src/components/RuleCard/index.jsx [hunk 2]
```diff
@@ -152,6 +152,7 @@ function RuleCard({
   rule,
```
Comment (bhearsum) [inline ui/src/components/RuleCard/index.jsx:152]: The changes to this file look unrelated to the issue you're fixing. Are they supposed to be in this PR? (If not, please revert them.)

File: ui/src/views/Rules/ListRules/index.jsx [hunk 3]
```diff
@@ -149,6 +149,7 @@ function ListRules(props) {
   const [dateTimePickerError, setDateTimePickerError] = useState(null);
```
Comment (bhearsum) [inline ui/src/views/Rules/ListRules/index.jsx:149]: And this one as well...it looks like this might be for a different issue?

**Review by bhearsum — CHANGES_REQUESTED** [review]
This is a good start! One overall thing I noticed is that the chips are often wider than they need to be, causing unnecessary wrapping. For example, in this screenshot the "Action Restrictions" ought to be able to fit on two lines instead of three:
image

File: ui/src/components/AutoCompleteText/index.jsx (hunk 1 above)
Comment (CREDO23) [reply ui/src/components/AutoCompleteText/index.jsx:190]: I added this class to change the cursor behavior (pointer) when hovering the endAdornment for `TextField` (the dropDown Icon) to indicate that it is a clickable element.
```javascript
    endAdornment: {
    cursor: 'pointer',
    },
```
A visual difference can be seen when hovering the dropDown icon

@bhearsum

File: ui/src/components/RuleCard/index.jsx (hunk 2 above)
Comment (CREDO23) [reply ui/src/components/RuleCard/index.jsx:152]: Changes to this file are related to previous PRs ( [PR 1](https://github.com/mozilla-releng/balrog/pull/2999), [PR 2](https://github.com/mozilla-releng/balrog/pull/3002), ) I made, approved but not merged !

@bhearsum

File: ui/src/views/Rules/ListRules/index.jsx (hunk 3 above)
Comment (CREDO23) [reply ui/src/views/Rules/ListRules/index.jsx:149]: Changes to this file are related to previous PRs ( https://github.com/mozilla-releng/balrog/pull/2999, https://github.com/mozilla-releng/balrog/pull/3002, ) I made, approved but not merged !

@bhearsum

File: ui/src/components/RuleCard/index.jsx (hunk 2 above)
Comment (bhearsum) [reply ui/src/components/RuleCard/index.jsx:152]: Please revert them from this branch, in that case. We cannot merge this PR with unrelated changes included.

Comment (CREDO23) [reply ui/src/components/RuleCard/index.jsx:152]: I understand! Let me do it!

Comment (CREDO23) [reply ui/src/components/RuleCard/index.jsx]: @bhearsum, I have just reverted unrelated changes!

File: ui/src/components/AutoCompleteText/index.jsx (hunk 1 above)
Comment (bhearsum) [reply ui/src/components/AutoCompleteText/index.jsx:190]: Ah, thank you! I overlooked that.

**Review by bhearsum — COMMENTED** [review]
Once https://github.com/mozilla-releng/balrog/pull/3024#pullrequestreview-1694840876 is addressed this should be ready to go.

**Review by bhearsum — APPROVED** [review]
Looks good - thank you!

**Pull request merged**